ambry-router/src/main/java/com.github.ambry.router/NonBlockingRouterMetrics.java

```diff
@@ -277,6 +298,21 @@ public Integer getValue() {
   }
 
   /**
+   * Initializes a {@link Gauge} metric for the maxPutChunkSize. This value is a constant for a running router
+   * instance, but can change between invocations. This metric helps maintain historical information.
+   * @param maxPutChunkSize the value to initialize this metric with.
+   */
+  public void initializeMaxPutChunkSizeMetric(final int maxPutChunkSize) {
+    maxPutChunkSizeGauge = new Gauge<Integer>() {
+      @Override
+      public Integer getValue() {
+        return maxPutChunkSize;
+      }
+    };
+    metricRegistry.register(MetricRegistry.name(NonBlockingRouter.class, "MaxPutChunkSizeBytes"), maxPutChunkSizeGauge);
```

Comment:
I don't understand the purpose of this metric - the configuration should be enough to indicate this (in that sense it isn't even a metric - what are we measuring?). Why have a gauge that exports the same number over and over again - something that has _absolutely_ no chance of changing?